cocotb/share/lib/embed/gpi_embed.cpp

```diff
@@ -118,10 +118,6 @@ extern "C" void embed_init_python(void)
 #define PY_SO_LIB xstr(PYTHON_SO_LIB)
 #endif
 
-    // Don't initialize Python if already running
-    if (gtstate)
-        return;
```

Comment:
Perhaps safest would be `assert(!gtstate, "embed_init_python should not be called twice")`